proto/context/src/main/proto/spine/validation/field_views.proto

```diff
@@ -11,17 +11,30 @@ option java_multiple_files = true;
 
 import "spine/protodata/ast.proto";
 
+// Uniquely identifies a message field.
+message FieldId {
+
+    // Name of the containing type.
+    protodata.TypeName type = 1;
+
+    // Short name of the field.
```

Comment:
I think it's enough that we call the field name "short" in the declaration of the `FieldName` itself.